### DISCUSSION TITLE
What does the name(bear) exactly mean?

### DISCUSSION THREAD

**Discussion by leleliu008** [rizsotto/Bear#520] [Q&A]
document says: `Bear(Build EAR) is a tool that generates a compilation database for clang tooling.`

I am curious about the relationship between `EAR` and `clang compilation database`. Thanks.

**Comment by rizsotto** [comment]
Projects needs names. Bear is just a name for this project. There is no deep message encoded in the name... I like to choose names which are not specific to the job the tool (or service) is doing, because the function might change overtime. So, picking a name like "Jenkins" for a continues integration service is great (instead of calling "continuous integration service").

With bear, I was playing with words. It can be interpreted just a name as "bear", but if people want to connect it to the functionality it can be "build ear" because it listens to your build (and generates compilation database as output).

**Reply by leleliu008** [reply]
Thank you for the explanation.
